test/common.dart

```diff
@@ -146,9 +146,10 @@ Future wait(int milliseconds) {
   return new Future.delayed(new Duration(milliseconds: milliseconds), () {});
 }
 
-Future requestAnimationFrame(){
+Future requestAnimationFrame([function (double)]){
```

Comment:
Lets change `function` => `callback`, add `void` as the return type, and give the double a name just so its more clear what it represents:

```dart
Future requestAnimationFrame([void callback(double elapsed)]) {
```